### PR TITLE
FWF-3752 [Bugfix] : Changes as part of css optimisations initial commit

### PR DESCRIPTION
**Pull request by arun-s-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3752
Issue Type: BUG

# Changes

- Rework on modal sizing
- Removal of unused styles
- Removal of default variables
- Convertion of styles to variables
- Duplicate code removal
- Standardising css to be compatible to formio
- Nesting of styles

# Notes

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/CustomComponents/FormInput.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ const FormInput = ({
 
   const inputClassNames = "form-control-input " + className ;
   return (
-    <div className="form-input-box" >
       <Form.Group controlId={id}>
       {label && (
```
Comment (fahad-aot) [inline forms-flow-web/src/components/CustomComponents/FormInput.js:27]: This component is removed from open source in last PR .

File: forms-flow-web/src/components/CustomComponents/settingsModal.js [hunk 2]
```diff
@@ -461,7 +461,7 @@ const SettingsModal =
         
         
       </Modal.Body>
-      <Modal.Footer className="d-flex justify-content-start">
+      <Modal.Footer>
```
Comment (Bonymol-aot) [inline forms-flow-web/src/components/CustomComponents/settingsModal.js:464]: Had checked any breakages by removing this class?

Comment (arun-s-aot) [reply forms-flow-web/src/components/CustomComponents/settingsModal.js:464]: Yes. handled in generic modal footer class @Bonymol-aot

File: forms-flow-web/src/components/CustomComponents/FormInput.js (hunk 1 above)
Comment (arun-s-aot) [reply forms-flow-web/src/components/CustomComponents/FormInput.js:27]: Ok . will update the PR

**Review by fahad-aot — APPROVED** [review]

**Pull request merged**